samples/showcase/src/main/kotlin/org/jetbrains/jewel/samples/showcase/components/Links.kt

```diff
@@ -54,10 +50,17 @@ fun Links() {
     Row(horizontalArrangement = Arrangement.spacedBy(10.dp), verticalAlignment = Alignment.CenterVertically) {
         Link("Link", {}, enabled = false)
 
-        Link("Always underlined", {}, style = alwaysUnderlinedStyle, enabled = false)
+        Link("Always underlined", {}, style = JewelTheme.linkStyle, enabled = false)
```

Comment:
```suggestion
        Link("Always underlined", onClick = {}, enabled = false)
```

That style is the default... So assuming it's correct, no need to specify it.

But, in the "before" state, this was `alwaysUnderlinedState`, so not sure this is correct